fe/fe-core/src/main/java/com/starrocks/catalog/IcebergResource.java

```diff
@@ -64,6 +68,17 @@ protected void setProperties(Map<String, String> properties) throws DdlException
                     throw new DdlException(ICEBERG_METASTORE_URIS + " must be set in properties");
                 }
                 break;
+            case CUSTOM_CATALOG:
+                catalogImpl = properties.get(ICEBERG_IMPL);
+                if (StringUtils.isBlank(catalogImpl)) {
+                    throw new DdlException(ICEBERG_IMPL + " must be set in properties");
+                }
+                try {
+                    Thread.currentThread().getContextClassLoader().loadClass(catalogImpl);
```

Comment:
Am I missing some logic? I think loadClass only occurs when the resource is created. But when the fe process is restarted, the custom class doesn't seem to be loaded into the jvm.